src/app/users/UserHome.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
+import ViewChefSchedule from './ViewChefSchedule';
 
-const UserHome = () => <div>User Home View Here</div>;
+const UserHome = () => <div><ViewChefSchedule /></div>;
```

Comment:
Do you want to put this component somewhere else? Or do you mind if I do when I PR?

Reply:
Yeah, I just stuck it there to be able to view it and play around with it a bit. You can move it wherever makes sense.